### PR TITLE
fix: handle fal keyword being empty in schema.yml

### PR DESCRIPTION
**Pull request by chamini2**
<details>
closes FEA-188
</details>

### CODE REVIEW

File: src/faldbt/project.py [hunk 1]
```diff
@@ -365,6 +365,7 @@ def _find_features(self) -> List[Feature]:
             filter(
                 # Find models that have both feature store and column defs
                 lambda model: keyword in model.meta
+                and isinstance(model.meta[keyword], dict)
```
Comment (mederka) [inline src/faldbt/project.py:368]: This is a good one. But what if the user passes a string by mistake? Shouldn't we let the user know?

Comment (chamini2) [reply src/faldbt/project.py:368]: Would you just raise an exception?

Comment (mederka) [reply src/faldbt/project.py:368]: Yes, I think so.

**Comment by chamini2** [comment]
@mederka , for now I am just checking what _needs_ to be checked for it to work, but I agree it is not super clean. I added a new issue FEA-191 to address it better down the road.

File: src/faldbt/project.py [hunk 2]
```diff
@@ -132,24 +132,44 @@ def get_depends_on_nodes(self) -> List[str]:
         return self.node.depends_on_nodes
 
     def get_post_hook_paths(self, keyword: str = "fal") -> List[str]:
-        return self.meta.get(keyword, {}).get("post-hook", [])
+        meta = self.meta or {}
+
+        keyword_dict = meta.get(keyword) or {}
+        if not isinstance(keyword_dict, dict):
+            return []
```
Comment (isidentical) [inline src/faldbt/project.py:139]: Question: what would be the case for `keyword_dict` being something else than `dict`? If it only happens when it is missing, maybe we could chain these and simplify the overall logic. 

Something like this maybe:
```python
return meta.get(keyword, []).get("post-hook", [])
```

**Pull request merged**